ERCS/erc-6538.md

```diff
@@ -109,9 +105,8 @@ You can find an implementation of this standard above.
 
 ## Security Considerations
 
-TODO
+In the event of a compromised private key, the registrant should promptly un-register from the stealth key registry to prevent loss of future funds sent to the compromised account.
```

Comment:
We should go into a bit more detail here because this is easier said than done—once compromised, any ETH sent gets drained by bots right away, so they have to do this with a signature and relayer, or use a private mempool like flashbots to batch the transactions if they chain supports it.

Reply:
Tried expanding in https://github.com/garyghayrat/ERCs/pull/2/commits/9d290de39729188aedaf951150e79e474594c907 but still some room for improvement?

Reply:
That works, though it sounds like the signature must go through a private mempool, so maybe tweak it to clarify that approach is a separate solution than the sig solution

Reply:
You're right, how about this: https://github.com/garyghayrat/ERCs/pull/2/commits/b7d9a58dfdc90095c2069b60d0cca557419e322d